Source/MSBuildTools.Unity/Assets/package.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "com.microsoft.msbuildforunity",
+    "displayName": "MSBuild for Unity",
+    "version": "0.0.2",
+    "unity": "2018.1",
+    "description": "A set of tools for Unity to make it simpler to work with MSBuild toolchain.",
+    "keywords": [
+        "MSBuild",
+		"NuGet",
+        "unity"
+    ],
+	"repository": {
+      "type": "git",
+      "url": "https://github.com/microsoft/MSBuildForUnity.git"
+	},
+	"files":["ProjectBuilder", "ProjectGenerator"],
```

Comment:
>"ProjectBuilder", "ProjectGenerator" [](start = 10, length = 36)

Ok, so we're packaging all of this together - one part for consuming MSBuild projects, one part for producing MSBuild projects. And then I guess if NuGetForUnity took a dependency on this package, it could help with both consuming and producing NuGet packages through these tools?

Reply:
Yeah, we can optionally split into separate packages, but I feel we should deliver as a single package right now.